index.html

```diff
@@ -14,34 +15,267 @@
-
+      
           Finkenau 35<br />
           22081 Hamburg
         </a>
+        <br />
+        <br />
+
+        <p>
+          <h4>Getting there</h4>
+
+          The easiest way to get there is by walking 500m from the U-Bahn (Underground) station “Mundsburg”, which is about a 7
+          Minute ride away from the central station (“Hauptbahnhof”).<br />
+          <br />
+          It’s probably not a good idea to try to go by car, as parking space is very limited and even on a saturday, we can’t
+          really recommend driving in Hamburgs inner city.<br />
+          <br />
+          If you are arriving from Hamburg Airport, you should take the S-Bahn (Metro) to Barmbek and then the U-Bahn to
+          Mundsburg.<br />
+          <br />
+
+          <h4>Surroundings</h4>
+
+          The Venue is right next to the Eilbekkanal, a channel that runs directly to the Alster lake (“Aussenalster”) which is
+          probably a 30 minute walk but especially worth it on a sunny spring day.
+        </p>
+      </div>
+
+      <div class="city">
+        <h3>## Hamburg</h3>
+
+        <p>
+          Quoting <a href="https://en.wikipedia.org/wiki/Hamburg">Wikipedia</a>:
+
+          <blockquote>
+            Hamburg, officially the Free and Hanseatic City of Hamburg, is the second-largest city in Germany after Berlin and 8th
+            largest city in the European Union with a population of over 1.8 million.
+          </blockquote>
+
+          Getting to Hamburg is easy. By bike, train, car, plane - even by boat.
+          The <a href="https://en.wikivoyage.org/wiki/Hamburg">Wikivoyage travel guide</a> sums up everything
+          you need to know about getting to and getting around Hamburg, finding a place to stay or even moving here.
+        </p>
+      </div>
+    </article>
+
+    <article id="inclusivity">
+      <div class="separator">
+        <img src="assets/sticker/07.png" />
+        <img src="assets/sticker/08.png" />
+        <img src="assets/sticker/09.png" />
+        <img src="assets/sticker/10.png" />
+        <img src="assets/sticker/11.png" />
+        <img src="assets/sticker/12.png" />
+      </div>
+
+      <h2># Inclusivity</h2>
+
+      <div>
+        <h3>## Diversity</h3>
+
+        <p>
+          Our scholarship program is meant to make Ruby Unconf a better unconference. We want to make this community-focused event
+          accessible to the wide range of people who are learning, work with, or are Ruby enthusiasts. We welcome applicants from
+          underrepresented groups in tech to apply. This includes, but is not limited to: women-identifying persons, people of
+          color, LGBTQIA people, people with physical and mental disabilities and people facing economic and/or political
+          hardships. Scholarship recipients will receive a Ruby Unconf Hamburg 2020 ticket.
+        </p>
+
+        <p>
+          With the purchase of a diversity ticket you help us to make this happen.
```

Comment:
We don't have a "real" diversity ticket this time, so maybe: 

"With paying more than the recommended price of 65€ you help us make this happen"